paddle/fluid/operators/fused/fused_conv2d_op.cc

```diff
@@ -0,0 +1,164 @@
+/* Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include <string>
+#include <vector>
+
+#include "paddle/fluid/operators/conv_op.h"
+
+namespace paddle {
+namespace operators {
+
+class FusedConvOpMaker : public framework::OpProtoAndCheckerMaker {
```

Comment:
OpMaker Code of fused op is a superset of OpMaker of base op. Is it possible to make fused one to inherit from base one in order to avoid redundant code duplication?

Reply:
Ok, it's a good suggestion~